io_ogre/config.py

```diff
@@ -116,7 +117,8 @@ def load_config():
             with open( CONFIG_FILEPATH, 'rb' ) as f:
                 config_dict = pickle.load( f )
         except:
-            print('[ERROR]: Can not read config from %s' %CONFIG_FILEPATH)
+            #print('[ERROR]: Can not read config from %s' %CONFIG_FILEPATH)
```

Comment:
please remove the `print`s instead of merely commenting them out. There is no purpose in keeping them around.

Reply:
Done